src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -50,6 +50,14 @@ Public Module BuildDevDivInsertionFiles
             "codeAnalysisService.servicehub.service.json",
             "remoteHostService.servicehub.service.json",
             "serviceHubSnapshotService.servicehub.service.json",
+            "Microsoft.Build.Conversion.Core.dll",
+            "Microsoft.Build.dll",
+            "Microsoft.Build.Engine.dll",
+            "Microsoft.Build.Framework.dll",
+            "Microsoft.Build.Tasks.Core.dll",
+            "Microsoft.Build.Utilities.Core.dll",
+            "Microsoft.VisualStudio.Threading.dll",
+            "Microsoft.VisualStudio.Validation.dll",
```

Comment:
FYI. I talked with @srivatsn about this before going this route.

anyway, for 

1) legacy set up VS will continue to work. just OOP won't be supported. as far as I know and told, since Dev15 is willow only and OOP is for dev15, OOP not working with legacy setup is okay. (service hub won't work with dev14 and dev15 preview 3 - for preview 3, if one override preview 3 service hub with preview 4 service hub bits, it will work)

2) MS.VS.* dlls are required for JsonRpc we use as a mechanism to communicate with VS. if we change it to some other mechanism such as MessagePack or others, we will need dlls for those. basically, it is just part of lib we use like NewstonJson dll. we just happen to not needed to distribute it in preview 3 since service hub.exe itself used that and they  were already loaded. which is no longer case for preview 4 since service hub now isolate us from them so we no longer share those dlls.

Microsoft.Build.* dependency is not specifically for OOP, but MS.CA.Workspace.Desktop's  runtime dependencies. any one who consume MS.CA.Workspace.Desktop.dll will need them (which our service hub services does).

we can get rid of those if we split MSBuildWorkspace out from MS.CA.Workspace.Desktop.dll by the way. (like MS.CA.MSBuildWorkspace.dll which has dependency to MS.CA.Workspace.Desktop.dll)